### PR TITLE
#29 - Add -upgrade option to init action

### PR DESCRIPTION
**Pull request by kingsleyadam**
Bump version and add to changelog

### CODE REVIEW

File: tests/test_action_init.py [hunk 1]
```diff
@@ -20,6 +21,7 @@ def test_run(self, mock_init, mock_check_result, mock_chdir):
         test_plan_path = "/terraform"
         test_terraform_exec = "/usr/bin/terraform"
         test_backend = {'path': '/terraform/terraform.tfstate'}
+        test_upgrade = True
```
Comment (nmaludy) [inline tests/test_action_init.py:63]: you should probably have some extra tests in here that passes in `None` and `False` for the `upgraded` to ensure proper behavior.

The idea here is the we want to verify the code works under all circumstances

**Review by nmaludy — CHANGES_REQUESTED** [review]
One small change in the tests

**Review by nmaludy — APPROVED** [review]
LGTM!
